kitsune/sumo/utils.py

```diff
@@ -350,3 +351,37 @@ def check_for_spam_content(data):
     has_links = has_blocked_link(data)
 
     return is_toll_free or is_nanp_number or has_links
+
+
+WEBPACK_SHORT_PATH_MAPPING = {
+    "protocol/": "node_modules/@mozilla-protocol/core/protocol/",
+    **{
+        f"{app}/": f"kitsune/{app}/static/{app}/"
+        for app in [
+            app.removeprefix("kitsune.")
+            for app in settings.INSTALLED_APPS
+            if app.startswith("kitsune.")
+        ]
+    },
+}
+_webpack_static_cache = {}
+
+
+def webpack_static(source_path):
+    """
+    Get the URL for an asset processed by webpack.
+    Accepts a shortened path to the source, like:
+    "sumo/img/mozilla-support.svg" rather than "kitsune/sumo/static/sumo/img/mozilla-support.svg"
+    """
+
+    if settings.WEBPACK_USE_STATIC_CACHE and source_path in _webpack_static_cache:
+        return _webpack_static_cache[source_path]
+
+    with open("./dist/source-to-asset.json") as f:
+        source_to_asset = json.load(f)
+        full_path = re.sub(
```

Comment:
This feels redundant. We already have the asset under the `django_static(source_path)`. I will continue looking into this though